test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -76,6 +76,28 @@ protected override async Task Test(CancellationToken cancellationToken)
                 }
             }
 
+            // The following metric should not be populated in a happy E2E path.
+            // We are going to make a list and remove them here to not consider them as a failure.
+            IEnumerable<string> acceptableDegeneratedMetrics = new HashSet<string>
+            {
+                // "edgeagent_direct_method_invocations_count", // BEARWASHERE -- Actually make this pass
+                "edgeAgent_metadata",
```

Comment:
This one (`edgeAgent_metadata`) should already be showing up in the metrics. Maybe it's just a case mismatch or something silly like that? @lfitchett

Reply:
From the most recent test run, it seems like this metrics was not populated. 
Per \iotedge\doc\BuiltInMetrics.md, it seems like such matric can possibly not being populated if none of the flags mentioned are set.

Reply:
It should definitely be showing up. That metric is always on and we definitely want to verify it shows up